### PR TITLE
Feature/bulk case upload

### PR DESCRIPTION
**Pull request by WeiWu2**
### **Changes:**
- new endpoints for getting cases by user_id and removing a pending case.
- swagger comment clean up.
- added Date to uploaded cases in MM/DD/YYYY format.
- removed status and upload from case table and seeds, all cases in the cases table will be only approved so status is obsolete along with upload (upload might be added back later).

### CODE REVIEW

File: api/upload/uploadRouter.js [hunk 1]
```diff
@@ -89,6 +88,9 @@ router.post('/', authRequired, (req, res) => {
               filed_in_one_year: false,
               credible: false,
               status: 'pending',
+              uploaded: `${
+                uploadedDate.getMonth() + 1
+              }-${uploadedDate.getDate()}-${uploadedDate.getFullYear()}`,
```
Comment (MichelleCCodes) [inline api/upload/uploadRouter.js:93]: Nice reformatting of the date!! This change improves the user experience greatly. Thanks for that!

**Review by MichelleCCodes — APPROVED** [review]
Good work on cleaning up the excess code as you're working through these files in addition to restructuring the migrations to reflect change in status/upload feature!

File: api/pendingCases/pendingCaseModel.js [hunk 2]
```diff
@@ -23,8 +23,8 @@ const add = async (data) => {
 //   return await db('cases').where({ primary_key: id });
 // };
 
-const remove = async (id) => {
-  return await db('pending_cases').where({ primary_key: id }).del();
+const remove = async (pending_case_id) => {
```
Comment (paulstgermain) [inline api/pendingCases/pendingCaseModel.js:26]: Loving the more specific naming conventions here!

File: api/upload/uploadRouter.js (hunk 1 above)
Comment (paulstgermain) [reply api/upload/uploadRouter.js:93]: Nothing like properly formatted dates to orient the users - thanks for this change!

**Review by paulstgermain — APPROVED** [review]
Nice changes here, the code is concise and effective, and I appreciate the cleanup and reformatting that went into this - approved!

**Pull request merged**